Library/Homebrew/language/node.rb

```diff
@@ -4,13 +4,21 @@ def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache\n"
     end
 
-    def self.pack_for_installation
+    def self.pack_for_installation(prepublishRequiresDeps)
+      # Some packages are requiring (dev)dependencies to be in place when
+      # running the prepublish script (which is run before pack). In this case
+      # we have to install all dependencies a first time already before
+      # executing npm pack (and a second time when doing the actual install).
+      if prepublishRequiresDeps
+        system "npm", "install" # ToDo: check exit status
```

Comment:
Can someone give me a hint about how I handle exit codes here? (looks like it doesn't raise automatically here like in formulas)